helm/app-exporter/templates/configmap.yaml

```diff
@@ -10,25 +10,18 @@ data:
     server:
       enable:
         debug:
-          server: true
+          server: {{ .Values.config.debug }}
       listen:
-        address: 'http://0.0.0.0:8000'
+        address: http://0.0.0.0:{{ .Values.config.listenPort }}
     service:
       collector:
         apps:
           # appTeamMappings can be used when the team annotation is missing in Chart.yaml.
           # Make sure you also add the missing annotation.
-          appTeamMappings: |
-            efk-stack-app: "atlas"
-          defaultTeam: "honeybadger"
+          appTeamMappings: | {{ nindent 12 .Values.config.appTeamMappings }}
```

Comment:
Will not this end up as invalid yaml? I mean it will add the first line after the pipe, like this:

```yaml
b: | this
  is
  multiline
```

that is invalid yaml, the first line must start on the next line.

Reply:
Or maybe you did not intended it to be a multi line string.

Reply:
`nindent` is "new line and indent", as compared to just `indent`. So this renders fine.